src/pages/dar_application/ProgressReportApplication.tsx

```diff
@@ -31,8 +33,10 @@ type ProgressReportApplicationProps = {
 };
 
 export const ProgressReportApplication = ({ dar, datasets, readOnlyMode = true, history, location, researcher, countriesOfOperation }: ProgressReportApplicationProps) => {
-    const initialState = {
+    const initialState: FormState = {
         ...dar,
+        publications: (dar.publications || []) as unknown as PublicationOrPresentation[],
```

Comment:
I also added some changes around this in https://github.com/DataBiosphere/duos-ui/pull/2968.  Do you know why we're typing a Publication as something other than a Publication?

Reply:
These were typed as `PublicationOrPresentation[]` a few weeks ago without the intermediate `unknown`.  I've incorporated the more direct `Publication[]` type.